src/lower.c

```diff
@@ -5812,7 +5861,7 @@ static void lower_case_scalar(tree_t stmt, loop_stack_t *loops, ident_t hier)
       } else {
          vcode_select_block(start_bb);
          cases[cptr]  = lower_reify_expr(tree_name(a));
-         
+
```

Comment:
BTW you can probably configure your editor to automatically strip trailing whitespace like this.

Reply:
I set-up my IDE to do that. So far I wasn't really careful about this, but its always good to improve :)